Core/clim-core/panes/stream-panes.lisp

```diff
@@ -365,49 +365,50 @@ current background message was set."))
 ;;;
 ;;; CONSTRUCTORS
 ;;;
-
 (defun make-clim-stream-pane (&rest options
                               &key (type 'clim-stream-pane)
+                                label
+                                label-alignment
+                                (display-after-commands nil display-after-commands-p)
                                 (scroll-bar :vertical)
                                 (scroll-bars scroll-bar)
                                 (borders t)
                               &allow-other-keys)
-  (with-keywords-removed (options (:type :scroll-bar :scroll-bars :borders))
-    ;; The user space requirement options belong to the scroller ..
-    (let* ((space-keys '(:width :height :max-width :max-height
-                         :min-width :min-height))
-           (user-sr nil)
-           (pane-options nil))
-      (loop  for (key value) on options by #'cddr
-             if (and (member key space-keys :test #'eq)
-                     (not (eq value :compute)))
-               nconc (list key value) into space-options
-             else
-               nconc (list key value) into other-options
-             end
-             finally (progn
-                       (setq user-sr space-options)
-                       (setq pane-options other-options)))
-      (let* ((pane (apply #'make-pane type (append pane-options
-                                                   (unless (or scroll-bars
-                                                               borders)
-                                                     user-sr))))
-             (stream pane))
-        (when scroll-bars
-          (setq pane (apply #'make-pane 'scroller-pane
-                            :scroll-bar scroll-bars
-                            :contents (list (make-pane 'viewport-pane
-                                                       :contents (list pane)))
-                            (unless borders
-                              user-sr))))
-        (when borders
-          (setq pane (apply #'make-pane 'border-pane
-                            :border-width (if (not (numberp borders))
-                                              1
-                                              borders)
-                            :contents (list pane)
-                            user-sr)))
-        (values pane stream)))))
+  (with-keywords-removed (options (:type :scroll-bar :scroll-bars :borders :label
+                                         :label-alignment :display-after-commands))
+    ;; See Spec 29.4.3 for :display-after-commands keyword.
+    (when (and display-after-commands-p (not (member :display-time options)))
+      (if (eq t display-after-commands)
+          (nconc options (list :display-time :command-loop))
+          (nconc options (list :display-time display-after-commands))))
```

Comment:
The fact that the result of `nconc` isn't used seems wrong.

How about this?
```lisp
(let* ((other-options (alexandria:remove-from-plist options :type …))
       ;; See Spec 29.4.3 for :display-after-commands keyword.
       (final-options (if (or (member :display-time options) (not display-after-commands-p))
                          other-options
                          (list* :display-time
                                 (if (eq display-after-commands t) 
                                     :command-loop 
                                     display-after-commands)
                                 other-options))
   …)

Reply:
I have used your code with the difference that because now an error is signaled if both `:display-after-commands` and `:display-time` are supplied I can trust that if `display-after-command-p` is true than there isn't `:display-time` in the options